examples/jsm/loaders/NRRDLoader.js

```diff
@@ -457,12 +458,12 @@ class NRRDLoader extends Loader {
 
 			const v = headerObject.vectors;
 
-			const ijk_to_transition = new Matrix4().set(
+			const ijk_to_transition = new Matrix4().fromArray([
```

Comment:
The `set()` method takes arguments in row-major order. `fromArray()` uses column-major. It seems without reordering the arguments the resulting matrix is now incorrect.

Reply:
Thanks so much for pointing out this. I will undo this part. 
I tested it in different nrrd cases, the result looks the same. 
But I think I should follow your suggestions, just in case there may any potential bugs.